src/consensus/tx_verify.cpp

```diff
@@ -135,9 +136,9 @@ unsigned int GetP2SHSigOpCount(const CTransaction& tx, const CCoinsViewCache& in
     return nSigOps;
 }
 
-int64_t GetTransactionSigOps(const CTransaction& tx, const CCoinsViewCache& inputs, int flags)
+int32_t GetTransactionSigOps(const CTransaction& tx, const CCoinsViewCache& inputs, int flags)
```

Comment:
What are the reasons for this type of change?

Reply:
Our max block size xfield is only 4 bytes.  Transaction Sigops, block signups are all derived from it. So I made them all 4 bytes.